### ISSUE TITLE
Can't view any models:  ENOENT error no such file or directory when the Prisma schema filename is not `schema.prisma`

### ISSUE THREAD

**Issue by huzaifahj** [prisma/studio#819]
Prisma version: 3.8.0-dev.27 (studio: 0.449.0)

Does the issue persist even after updating to the latest `prisma` alpha? (`npm i -D prisma@dev`): Yes

### Logs

Console:
```
npm run prisma-studio

> prisma-studio
> prisma studio -b none --schema schema1.prisma

Environment variables loaded from .env
Prisma schema loaded from schema1.prisma
Prisma Studio is up on http://localhost:5555
```
Prisma Studio Frontend:
```
[runQuery] Error in Prisma Client request: 

ENOENT: no such file or directory, open '/[...]/prisma/schema.prisma' 
Object
code:
args: {take: 100, skip: 0, select: {…}}
modelName: "user"
operation: "findMany"
[[Prototype]]: Object
error:
code: "ENOENT"
message: "ENOENT: no such file or directory, open '/[...]/prisma/schema.prisma'"
```

### Comments
I'm not sure why it's trying to load `schema.prisma` when the file I used to generate the client was `schema1.prisma`.

**Comment by eemmiillyy** [comment]
Hey @huzaifahj! Thanks for reporting this. We were able to reproduce it. We are working on a fix and will let you know as soon as it's released :)

**Comment by Jolg42** [comment]
@huzaifahj The current workaround would be to have the name of your Prisma schema file as `schema.prisma`

The location can be in any folder but this bug means than the filename can only be `schema.prisma` at the moment.

**Comment by huzaifahj** [comment]
@Jolg42 @eemmiillyy Thanks guys! That bug fix was very swift. Which NPM package version has this fix? Currently on the latest dev version but no luck

**Comment by eemmiillyy** [comment]
Hey @huzaifahj, that PR above was only the first part of the fix, we still need to make a change to Studio itself. Finally we'll need to get the Studio changes into the Prisma CLI at which point it will be 100% fixed.  This shouldn't take longer than a week or two though :)

**Comment by huzaifahj** [comment]
Thanks @eemmiillyy. Feel free to close this issue once 100% fixed

**Comment by eemmiillyy** [comment]
Hey @huzaifahj, this should be fixed as of `3.9.0-dev.7`. You can try this package in development, but we do not recommend using dev versions in production. You can also wait for the fix to be included in the next stable prisma release (`3.9.0`).

**Comment by huzaifahj** [comment]
Thanks @eemmiillyy !
